docs/gen-docs/kyma.md

```diff
@@ -15,9 +15,9 @@ Kyma CLI allows you to install, test, and manage Kyma.
 
 ```bash
       --ci                  Enables the CI mode to run on CI/CD systems. It avoids any user interaction (such as no dialog prompts) and ensures that logs are formatted properly in log files (such as no spinners for CLI steps).
-  -h, --help                Displays help for the command.
-      --kubeconfig string   Specifies the path to the kubeconfig file. By default, Kyma CLI uses the KUBECONFIG environment variable or "/$HOME/.kube/config" if the variable is not set.
-      --non-interactive     Enables the non-interactive shell mode.
+  -h, --help                See help for the command
```

Comment:
Is there a reason why some commands end with a `.` but some don't? 🤔 It would be better to make it consistent (e.g. remove the dots everywhere, but GitHub doesn't let me add suggestions to things that weren't changed in this PR).

Reply:
Thanks for spotting this - must have slipped through during several iterations of rewriting :)